### ISSUE TITLE
disabled={true} doesn't work

### ISSUE THREAD

**Issue by lightninglu10** [Fauntleroy/react-simple-dropdown#46]
Hey, I'm trying to use disabled={true} but it's not working. My component is still triggering and the menu pops. I just updated to the newest NPM package.

```
<Dropdown className={css(this.props.buttonStyle)} ref='dropdown' disabled={true}>
```

**Comment by Fauntleroy** [comment]
Are you using a custom `onClick` handler on the `DropdownTrigger` component?

**Comment by lightninglu10** [comment]
@Fauntleroy no, but i'm looking in the node_modules package and it doesn't have the disabled code in there. Have you published the new version to NPM?

**Comment by Fauntleroy** [comment]
@lightninglu10 You're right! Looks like this is still in an alpha release. Feel free to use `3.2.0-alpha.2` for now. I'll make a change to prevent `onClick` from firing and release it as `3.2.0` sometime soon (likely later today).

**Comment by lightninglu10** [comment]
Great! Look forward to it. Thanks for the awesome package.

**Comment by lightninglu10** [comment]
@Fauntleroy Have you released `3.2.0` yet?

**Comment by Fauntleroy** [comment]
@lightninglu10 `3.2.0` is now live: https://www.npmjs.com/package/react-simple-dropdown

Let me know if you have any issues!
